Training Modules/01 Introduction/1.02 Python If-Else/solution.py

```diff
@@ -1,9 +1,9 @@
 n = int(input().strip())
 if n%2 == 1:
     print("Weird")
-elif n>1 and n<6:
+If n>1 and n<6:
```

Comment:
The syntax here is wrong and this on the other hand does not solve [problem 1.2](https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Training%20Modules/01%20Introduction/1.02%20Python%20If-Else) in the repo.